webapp/src/editortoolbar.tsx

```diff
@@ -266,7 +265,7 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
         const targetTheme = pxt.appTarget.appTheme;
         const { compiling, isSaving } = this.props.parent.state;
         const { compileState } = this.state;
-        const compileTooltip = lf("Download your code to the {0}", targetTheme.boardName);
+        const compileTooltip = lf("Download your code to {0}", targetTheme.boardName);
```

Comment:
does this read worse for boards like the micro:bit? should we make this some sort of option?

Reply:
Are strings in pxtarget.json localizable? If so, I could override the entire tooltip.

Reply:
reverting for now.